modules/config-aws-access.adoc

```diff
@@ -55,7 +55,7 @@ It is not recommended to set a permissions boundary.
 .. Click *Create user*, a success page appears.
 .. Gather the IAM user’s Amazon Resource Name (ARN). The ARN will have the following format: `arn:aws:iam::000111222333:user/username`. Click *Close*.
 
-. Open the {console-redhat-com} in your browser and select the cluster you want to allow AWS infrastructure access.
+. Open the {cluster-manager-url} in your browser and select the cluster you want to allow AWS infrastructure access.
```

Comment:
Remove "the", or might be better changed to "Navigate to OpenShift Cluster Manager" or "In OpenShift Cluster Manager, select the cluster...."